Sprint-3/revise/implement/is-prime.test.js

```diff
@@ -1,3 +1,33 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
 // Then it should check if the num is prime
+
+function isPrime (num){
+    if (num <= 1){
+        return false
+    } else if (num === 2){
+        return true 
+    } else if ((num > 2)) {
+        let squareRootNum = Math.sqrt(num)
+        for (let i = 2 ; i <= squareRootNum; i++ ) {
```

Comment:
Can further speed up the performance in the following manner:
- return false if `num` is an even number
- check only odd divisors in the loop